packages/storage/src/indexed-db/index.ts

```diff
@@ -824,4 +812,25 @@ export class IndexedDb implements IndexedDbInterface {
       output: Value.fromJson(result.output),
     };
   }
+
+  fetchStakingTokenId(): AssetId {
+    const registryClient = new ChainRegistryClient();
+    const registry = registryClient.get(this.chainId);
+    const stakingToken = registry.stakingAssetId;
+
+    return stakingToken;
+  }
```

Comment:
note that you can just use `this.stakingTokenAssetId`

Reply:
simplified to using `this.stakingTokenAssetId` defined in the constructor